app/assets/stylesheets/modules/digital.scss

```diff
@@ -1,42 +1,163 @@
-
 /* Digital pack */
-.digital-cta {
-    padding-top: $gs-baseline*2;
-    padding-bottom: $gs-baseline*3;
+
+.digital-pack {
+    .page-header {
+        margin-bottom: $gs-baseline*2;
+    }
+}
+
+// CONTAINERS
+
+$left-column: gs-span(2);
+
+.digital-pack {
+    .gs-container {
```

Comment:
the digital pack page needs to look different at the moment, we'll be changing this page first and rollout the change gradually to the whole site.